<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Restaurant</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visitors:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Workers:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_visitors">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>220</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addPerson">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_workers">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>220</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_changePerson">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>105</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deletePerson">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>330</y>
      <width>105</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="menuList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>220</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="itemList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>220</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="changeEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu list:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menuitem details:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Visitor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Worker</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_prof">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Proffesion:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>State:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_surname">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Surname:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>380</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_surname">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_state">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>440</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prof">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="savePeopleButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getPeopleButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typecomboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FOOD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DRINK</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
